constants.go

```diff
@@ -47,6 +47,9 @@ var (
 
 	// ErrServerConfig - failed to start the DNSCrypt server - invalid configuration
 	ErrServerConfig = errors.New("invalid server configuration")
+
+	// ErrServerNotStarted - returned if there's nothing to shutdown
+	ErrServerNotStarted = errors.New("server is not started")
```

Comment:
In a future major version, consider using [constant errors].  Otherwise, it's rather ironic that the file `constants.go` contains variables :-) .

[constant errors]: https://dave.cheney.net/2016/04/07/constant-errors